app/components/grid-items.tsx

```diff
@@ -335,3 +375,95 @@ export default function GridItemsContainer({
     </section>
   );
 }
+
+type GridItemsSectionProps = {
+  id?: string;
+  className?: string;
+  title: string;
+  description?: string;
+  layout?: "horizontal" | "vertical";
+  descriptionStyle?: string;
+  children?: React.ReactNode;
+};
+
+export const GridItemsSection = ({
+  id,
+  className = "",
+  title,
+  description,
+  layout = "horizontal",
+  descriptionStyle,
+  children
+}: GridItemsSectionProps) => {
+  const isHorizontal = layout === "horizontal";
+  const hasBgClass = /\bbg-/.test(className);
+
+  const finalClass = cn("scroll-mt-20", className);
+
+  return (
+    <section
+      id={id}
+      className={finalClass}
+      style={
+        !hasBgClass
+          ? {
+              background:
+                "linear-gradient(0deg, #050013, #050013), linear-gradient(180deg, #1513374D 0%, #15133700 23.58%)"
+            }
+          : undefined
+      }>
+      <motion.div
+        className="mx-auto flex max-w-5xl flex-wrap items-center justify-center px-4 py-5 md:px-6 md:py-8"
+        initial={{ opacity: 0, y: 40 }}
+        whileInView={{ opacity: 1, y: 0 }}
+        transition={{ duration: 0.8, ease: "easeOut" }}
+        viewport={{ once: true, amount: 0.3 }}>
+        {description ? (
+          <motion.div
+            initial={{ opacity: 0, y: 20 }}
+            whileInView={{ opacity: 1, y: 0 }}
+            transition={{ delay: 0.2, duration: 0.6 }}
+            className={cn(
+              "w-full",
+              isHorizontal
+                ? "flex flex-col flex-wrap justify-between gap-4 md:flex-row"
+                : "flex flex-col md:items-center"
+            )}>
+            <h2
+              className={cn(
+                "flex-1 text-4xl font-medium leading-tight text-white md:text-[40px]",
+                isHorizontal ? "text-left" : "mb-4 text-left md:text-center"
+              )}>
+              {title}
+            </h2>
+            <p
+              className={cn(
+                "flex-1 text-base text-[#AFB0C3] md:text-lg",
+                isHorizontal
+                  ? "max-w-lg text-left"
+                  : `w-full text-left md:w-2/3 md:text-center xl:w-3/4 ${descriptionStyle}`
+              )}>
+              {description}
+            </p>
+          </motion.div>
+        ) : (
+          <motion.h1
+            initial={{ opacity: 0, y: 20 }}
+            whileInView={{ opacity: 1, y: 0 }}
+            transition={{ delay: 0.1, duration: 0.6 }}
+            className="w-full text-center text-4xl font-medium leading-tight text-white md:text-[40px]">
+            {title}
+          </motion.h1>
+        )}
+
+        <motion.div
+          initial={{ opacity: 0 }}
+          whileInView={{ opacity: 1 }}
+          transition={{ duration: 0.6 }}
```

Comment:
please add `viewport={{ once: true, amount: 0.3 }}` so that the animation doesn't run again when the component comes to view.